docs/getting-started/setup.md

```diff
@@ -0,0 +1,13 @@
+
+# Getting started: setup
+
+To use Reliability Kit you'll need a few things set up in advance.
+
+  1. A Node.js application (ideally built with [Express](https://expressjs.com/), but you should get some benefits even if you're not using it)
+
+  2. For your application to be running on Node.js 14 or above
+
```

Comment:
I would add the installation command and the quick description of the 2 packages